### DISCUSSION TITLE
Mouseover cannot be dismissed

### DISCUSSION THREAD

**Discussion by lornajane** [community/community#127571] [Accessibility] [answered]
### Select Topic Area

Question

### Body

I am a keyboard-only user, and I use GitHub in Firefox with [Vimium](https://vimium.github.io/).

When viewing a list of issues or pull requests, if I open one in a new window, a large modal pops up (I think it's the mouseover/hover thing), but I don't have a mouse and I cannot therefore "mouseout" and remove it. I must therefore instead reload the page each time I return to the list. As a team lead and project maintainer, spend a lot of time with lists of issues and pull requests, often while screensharing, and it's frustrating to be constantly slowed down and losing my place in the list. Is there a way I can dismiss the modal or stop it from coming up in the first place?

I started noticing this a few months ago and I hoped that it was just a passing glitch, but it's really getting in my way. Advice or workarounds would be deeply appreciated!

**Comment by leocaimmi** [comment]
I have the same problem

**Comment by ericwbailey** [comment]
@lornajane Thank you for reporting this. I'm a member of the GitHub accessibility team, and I'll be investigating this issue.

One thing that would be really helpful for determining the root cause is knowing the content the modal is showing. Would you mind sharing a screenshot, or the text content of the modal? The reason I ask is that I am currently unable to recreate the issue as reported from my test account.

**Comment by lornajane** [comment]
Thanks @ericwbailey , here's a screenshot. I asked someone else to try and it seems like escape does work for them, super frustrating that it doesn't work for me!
![Screenshot 2024-07-19 at 13-24-22 Pull requests · Redocly_redocly-cli](https://github.com/user-attachments/assets/243ca647-2830-4ef6-add5-88307968f00a)

**Reply by ericwbailey** [reply]
Thanks for including that image, it's really helpful for triaging this. I also wanted to say your feelings of frustration are definitely heard. 

I think there might be a few potential things that could be causing this. I'm going to do some investigation over on the code side of things and will hopefully have an answer sooner than later for you.

**Reply by Calinou** [reply]
The typical UX pattern is to press <kbd>Escape</kbd> to dismiss tooltips without having to use a mouse.

Note that this issue also occurs for user profile popups (e.g. hover @Calinou and try to press <kbd>Escape</kbd>).

**Reply by ericwbailey** [reply]
@lornajane I can confirm this bug on my end. Apologies for the slow reply, I got stuck traveling when Crowdstrike hit. Additionally, I believe there will be a fix for this as soon as next week.

@Calinou Thank you, and I am quite familiar with expected keyboard interactions for patterns like this. The fix should cover user profile popups, as well.

**Reply by ericwbailey** [reply, marked as answer]
@lornajane We have just released some updates that address the issue you've been experiencing. We refer to these modals as hovercards, and they can now:

- Be dismissed by pressing <kbd>Esc</kbd> on your keyboard, and also
- Completely turned off altogether via an account preference.

If you turn them off, you'll need to uncheck the "Hovercards" checkbox in [your Accessibility account settings](https://github.com/settings/accessibility), and then activate the "Save hovercard preferences" button.

If you'd like, you can also [read about the change on our blog](https://github.blog/changelog/2024-07-29-keyboard-navigation-improvements-for-hovercards/)

**Comment by lornajane** [comment]
Thank you very much @ericwbailey ! I checked the blog and learned how to open and close the hovercards from a focused element but with vimium, it clicks but doesn't focus, so I still can't close the popup hovercard. I've disabled them for now and I think that's going to improve my experience a lot - thank you so much for including the option to turn them off!!

**Reply by ericwbailey** [reply]
Ah, that's good to know—I'll pass that note along to our team for them to investigate.

I'm really glad we were able to get you a solution that works, and also thank you for your patience while we worked through the issue. 

As a side note, I also disabled them the second the feature was available 😅
